ingredient/ingredient-utils/src/functions.ts

```diff
@@ -92,10 +92,20 @@ export class CoreUtils extends BaseUtility {
         return await this.context.Ingredient.properties.source.valueAsync(this.context)
     }
 
-    public get_event_hub_namespace_name(pkgName: string, resourceGroup: string | null = null): string {        
+    public get_event_hub_namespace_name(pkgName: string): string {        
         const evenhHubNamespace = this.create_resource_name("ehn", pkgName, true);
        
         return `${evenhHubNamespace}`;
     }
+
+    public get_app_insights_name(name: string): string {
```

Comment:
Just noticing these. We shouldn't be adding resource specific helpers to coreutils? Shouldn't these live with their respected ingredients?

Reply:
I think it makes sense for them to be in their respective ingredient utils rather than common utils.  I can start by moving the ones I created.